Makefile

```diff
@@ -75,7 +75,7 @@ CREATE_TEMPLATE_ARGS+=--user "src/main.{cpp,c,cc}" --user "include/main.{hpp,h,h
 CREATE_TEMPLATE_ARGS+=--target v5
 CREATE_TEMPLATE_ARGS+=--output bin/monolith.bin --cold_output bin/cold.package.bin --hot_output bin/hot.package.bin --cold_addr 58720256 --hot_addr 125829120
 
-template:: patch_sdk_headers clean-template library
```

Comment:
are you sure this was a mistake? It has a semantically different meaning now.

Reply:
yeh, its just a kernel thing. If it overload it will duplicate a libpros.a in the project root directory.

Reply:
So the `template` rule in common.mk is no longer used then? Can it be removed?

Reply:
It's only being used by users. There's no template rule in the distributed makefile, only in common.mk.

Reply:
Ok, so the kernel build uses this `template` rule here, and user project builds use the one in `common.mk`? Then shouldn't the one in `common.mk` be moved to `template-Makefile`? If it's not used by both, it shouldnt be in `common.mk`, right?